examples/aws-mysql-data-seeding/README.md

```diff
@@ -0,0 +1,73 @@
+# Create mysql db with data. 
```

Comment:
Let's add two different instructions: one for a full example "based on the user's existing database" and a second that just references what's already committed for a quick example of how it works